packages/devtools_app/lib/src/shared/about_dialog.dart

```diff
@@ -32,9 +35,30 @@ class DevToolsAboutDialog extends StatelessWidget {
             children: [
               const Text('Encountered an issue? Let us know at '),
               _createFeedbackLink(context),
-              const Text('.')
+              const Text(','),
             ],
           ),
+          Wrap(
```

Comment:
Do we want to have a method _createDiscordLink, in consistency with _createFeedbackLink?

Reply:
refactored both to be stateless widgets.